Makefile

```diff
@@ -47,3 +47,28 @@ endif
 	@echo "Any other details can be found in the README."
 	@echo "Enjoy!"
 	@echo ""
+
+
+JQ := $(shell command -v jq 2> /dev/null)
+RNVERSION=$(shell cd node_modules/react-native && pod ipc spec React.podspec | jq '.version' -r)
+
+update_specs_repos:
+ifndef JQ
+		$(error "Please install jq before running `brew install jq`")
+endif
+
+	@echo "Updating Artsy specs repo";
+	# pod repo update artsy
```

Comment:
Was this meant to be left commented out?

Reply:
noooooppeee